README.md

```diff
@@ -8,7 +8,10 @@ for running backend
 
  
  for creating migration 
- ```go run create-migration.go -name {migrationName}```
+ ```go run emreddit_cmd.go create {migrationName}```
  
- for running migrations
- ```go run migration.go```
\ No newline at end of file
+ for running migrations up
+ ```go run emreddit_cmd.go up```
```

Comment:
these namings is not needed. you can convert back these. but you can also add a makefile command.